include/itkImageToTorchImageFilter.hxx

```diff
@@ -18,48 +18,76 @@
 #ifndef itkImageToTorchImageFilter_hxx
 #define itkImageToTorchImageFilter_hxx
 
-#include "itkImageToTorchImageFilter.h"
+#include <torch/torch.h>
 
-#include "itkImageRegionIterator.h"
-#include "itkImageRegionConstIterator.h"
+#include "itkImageToTorchImageFilter.h"
 
 namespace itk
 {
 
-template <typename TInputImage, typename TOutputImage>
-ImageToTorchImageFilter<TInputImage, TOutputImage>
-::ImageToTorchImageFilter()
-{}
+template< typename TInputImage, typename TOutputTorchImage >
+void
+ImageToTorchImageFilter< TInputImage, TOutputTorchImage >
+::PrintSelf( std::ostream &os, Indent indent ) const
+{
+  Superclass::PrintSelf( os, indent );
+}
+
+
+template< typename TInputImage, typename TOutputTorchImage >
+void
+ImageToTorchImageFilter< TInputImage, TOutputTorchImage >
+::GenerateInputRequestedRegion()
+{
+  // Is this correct?!!!  Instead, is the superclass implementation sufficient?!!!
+  // Call the superclass's implementation of this method
+  Superclass::GenerateInputRequestedRegion();
+
+  // Get pointers to the input image.
+  typename Self::InputImageType *inputImage = const_cast< typename Self::InputImageType * >( this->GetInput( 0 ) );
+
+  if( inputImage != nullptr )
```

Comment:
This should got into verify preconditions method.

Reply:
Will do.